docs/src/content/docs/components/card-grids.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Card Grids
+description: Learn how to wrap multiple cards in a grid in Starlight.
+sidebar:
+  order: 4
+---
+
+import { Card, CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+To wrap multiple [`<Card>`](/components/cards/) or [`<LinkCard>`](/components/link-cards/) components in a grid, use the`<CardGrid>` component.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<CardGrid slot="preview">
+	<Card title="Stars" icon="star">
+		Sirius, Vega, Betelgeuse
+	</Card>
+	<Card title="Moons" icon="moon">
+		Io, Europa, Ganymede
+	</Card>
+</CardGrid>
+
+</Preview>
+
+## Import
+
+```tsx
+import { CardGrid } from '@astrojs/starlight/components';
+```
+
+## Usage
+
+### Group cards
+
+Display multiple [`<Card>`](/components/cards/) components side-by-side when there’s enough space by grouping them using the `<CardGrid>` component.
+
+<Preview>
+
+```mdx {5,12}
```

Comment:
```suggestion
```mdx {5,8}
```
I think this is the intention? In the preview, only one line is highlighted and I wondered why!

Reply:
Great catch, thanks a lot, it was actually all examples in all formats having wrong lines/ranges since I removed the filenames.